src/applications/pre-need/definitions/address.js

```diff
@@ -285,33 +287,46 @@ export function uiSchema(
 
         const modifiedData = { ...formData };
 
-        if (
-          modifiedData.application &&
-          modifiedData.application.claimant &&
-          modifiedData.application.claimant.address &&
-          (modifiedData.application.claimant.address.country !== 'USA' ||
-            modifiedData.application.claimant.address.country !== 'CAN') &&
-          !modifiedData.application.claimant.address.state
-        ) {
-          modifiedData.application.claimant.address.state = '';
-        }
+        if (modifiedData.application) {
```

Comment:
Replaced repetitive if conditional with a single conditional with the non repeating parts inside of it. This conditional helps prevent undesired changes to the form data when nothing has been entered yet.